### DISCUSSION TITLE
Legal request transparency report

### DISCUSSION THREAD

**Discussion by clementbiron** [OpenTermsArchive/terms-types#38] [Qualified]
### Type name

Transparency report

### Alternative type names

Transparency statement
Government Requests

### Writer

service provider

### Audience

end user

### Object

legal demands to remove content or produce account information

### Examples

| Example | Service Name | URL | Title | Jurisdiction(s) |
|:----|:----|:----|:------|:-------------|
| 1 | Proton | [URL 🔗](https://proton.me/legal/transparency) | Transparency report | - |
| 2 | Threema | [URL 🔗](https://threema.ch/en/transparencyreport) | Transparency Report | - |
| 3 | Apple | [URL 🔗](https://www.apple.com/legal/transparency/) | Transparency Report | 🌐 |
| 4 | Apple | [URL 🔗](https://www.apple.com/legal/transparency/au.html) | Transparency Report Australia | 🇦🇺 |
| 5 | Meta | [URL 🔗](https://transparency.meta.com/reports/government-data-requests/) | Government Requests for User Data | 🌐 |
| 6 | Meta | [URL 🔗](https://transparency.meta.com/reports/government-data-requests/country/FR/) | Government Requests for User Data | 🇫🇷 |
| 7 | X | [URL 🔗](https://transparency.x.com/en/reports/information-requests#2021-jul-dec) | Information Requests | 🌐 |
| 8 | X | [URL 🔗](https://transparency.x.com/en/reports/countries/us#2021-jul-dec) | Information Requests | 🇺🇸 |
| 9 | X | [URL 🔗](https://transparency.x.com/en/reports/removal-requests#2021-jul-dec) | Removal Requests | 🌐 |
| 10 | Apple App Store | [URL 🔗](https://www.apple.com/legal/more-resources/docs/2023-App-Store-Transparency-Report.pdf) | App Store Transparency Report | 🌐 |

**Comment by MattiSG** [comment]
While this topic is extremely interesting, these documents do not constitute norms. They do not describe rights and duties for any user or provider of services.
While the DSA mandates transparency reports for VLOPS, which can be interpreted as users having a right to reading this content, Open Terms Archive does not aim at guaranteeing such rights, but at tracking the _sources_ of rights.

On the other hand, tracking such data and its consistency does go towards shifting the power balance from online platforms towards regulators and end users, which is the political goal of Open Terms Archive.

At a technical level, the Open Terms Archive engine is not the most appropriate for tracking such content, as it often is published in independent documents for each reporting period, meaning that manual updates will be needed on each period change.
It is also worth noting that a lack of inclusion of such a type in the default terms types does not mean that the Open Terms Archive engine will not be able to track such documents: it would simply mean that entities willing to use the engine for such tracking would have to create their own type for it.

At this stage, I am not in favour of adding this type. There are many documents that would be interesting to preserve and track over time, but that go beyond Open Terms Archive's focus. We should first consolidate Open Terms Archive in its current scope before expanding and enabling broader use cases for which the engine is not optimised.
